api/metadata/pilot/pilot.proto

```diff
@@ -52,6 +50,11 @@ service PilotService {
 
 	rpc RunCommandOnFrontgateNode (metadata.types.RunCommandOnFrontgateRequest) returns (metadata.types.String);
 	rpc RunCommandOnDrone (metadata.types.RunCommandOnDroneRequest) returns (metadata.types.String);
+}
 
+service PilotServiceForFG {
```

Comment:
It's not necessary to abbreviate, PilotServiceForFrontgate is ok.

Reply:
fixed